test/components/select/titles.test.js

```diff
@@ -15,22 +15,24 @@ describe("Select Screen - Titles", () => {
         mockText = { setOrigin: jest.fn() };
         mockScene = {
             scene: { key: "character-select" },
-            theme: {
-                titles: [
-                    { type: "image", key: "title", xOffset: 0, yOffset: -250 },
-                    { type: "text", value: "Select Your Character", xOffset: 0, yOffset: -260 },
-                    { type: "text", value: "Sub-Title", xOffset: 0, yOffset: -233 },
-                ],
-            },
             add: { image: jest.fn(() => mockImage), text: jest.fn(() => mockText) },
             assetPrefix: "character-select",
+            context: {
+                theme: {
+                    titles: [
+                        { type: "image", key: "title", xOffset: 0, yOffset: -250 },
```

Comment:
should this be called `offsetX` and `offsetY` instead?

Reply:
Yes. This is a prime example of what I was talking about. x and y would be better.

Reply:
🤦